{"name": "Проверяем создание двух курьеров", "status": "passed", "description": "Создаем два одинаковых курьера, проверяем что на второго падает ошибка", "start": 1721769779238, "stop": 1721769779910, "uuid": "3d060182-7f9b-44f6-80cb-8456a42f9e8c", "historyId": "b59e127749087fc52c96fd988baedd99", "testCaseId": "b59e127749087fc52c96fd988baedd99", "fullName": "tests.test_create.TestCreate#test_create_2_couriers", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create"}, {"name": "subSuite", "value": "TestCreate"}, {"name": "host", "value": "MagnusEquusPC"}, {"name": "thread", "value": "10728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create"}]}